.github/workflows/test.yml

```diff
@@ -0,0 +1,47 @@
+name: Test
```

Comment:
could we be more specific with the name?

Reply:
The intent is to make the `Test` workflow as container for different types of test jobs. The job names are descriptive.